### PR TITLE
Makes various things return qdel hints on initialization

### PR DESCRIPTION
**Pull request by chuga-git**
## What Does This PR Do
Found a few things not returning qdel hints while porting a more up to date ref tracking system. Also tries and somewhat fails to fix lightgeist harddels - at least its hud isn't hanging on anymore...

## Why It's Good For The Game
Not returning qdel hints is like not paying your taxes. In some cultures, this is considered bad. Morally, at least.

## Testing
Game didn't blow up on initialization. 

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

NPFC

### CODE REVIEW

**Review by Contrabang — APPROVED** [review]

File: code/game/objects/structures/aliens.dm [hunk 1]
```diff
@@ -435,7 +435,7 @@
 
 /obj/structure/alien/weeds/node/Initialize()
 	add_overlay("weednode")
-	..(loc, src)
+	. = ..(loc, src)
```
Comment (lewcc) [inline code/game/objects/structures/aliens.dm:438]:
```suggestion
	return ..(loc, src)
```

File: code/modules/clothing/spacesuits/ert_hardsuits.dm [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		var/mob/living/carbon/human/wearer = loc.loc	//loc is the hardsuit, so its loc is the wearer
 		if(ishuman(wearer))
 			register_camera(wearer)
-	..()
+	. = ..()
```
Comment (lewcc) [inline code/modules/clothing/spacesuits/ert_hardsuits.dm:29]:
```suggestion
	return ..()
```

File: code/modules/power/lights.dm [hunk 3]
```diff
@@ -247,11 +247,11 @@
 
 /obj/machinery/light/built/Initialize(mapload)
 	status = LIGHT_EMPTY
-	..()
+	. = ..()
```
Comment (lewcc) [inline code/modules/power/lights.dm:250]:
```suggestion
	return ..()
```

File: code/modules/power/lights.dm [hunk 4]
```diff
@@ -247,11 +247,11 @@
...
 
 /obj/machinery/light/small/built/Initialize(mapload)
 	status = LIGHT_EMPTY
-	..()
+	. = ..()
```
Comment (lewcc) [inline code/modules/power/lights.dm:254]:
```suggestion
	return ..()
```

**Review by lewcc — COMMENTED** [review]
you know i had to do it to em

**Review by lewcc — APPROVED** [review]

**Pull request merged**